ws-nextjs-app/pages/[service]/articles/handleArticleRoute.ts

```diff
@@ -111,7 +119,62 @@ export default async (context: GetServerSidePropsContext) => {
     mostRead = null,
     billboardCuration = null,
     mediaCuration = null,
-  } = secondaryData;
+  } = secondaryData || {};
+
+  let personalisedContent;
+
+  if (shouldAttemptPersonalisedTopicExperience) {
+    const countrySpecificTopics: Record<string, string> = {
+      ar: 'c7zp57yy6dzt',
+      cl: 'c340qyppkk8t',
+      mx: 'c340qyp6yggt',
+      co: 'c404v5gz1rkt',
+      es: 'c6vzy3wd189t',
+      ve: 'cpzd49v9rd1t',
+      us: 'cdr5613yzwqt',
+      uy: 'cpzd498zwj6t',
+      do: 'cr50y7pykkdt',
+    };
+
+    const countrySpecificId =
+      country && countrySpecificTopics[country.toString()];
+
+    if (countrySpecificId) {
+      try {
+        const countrySpecificData = await fetchDataFromBFF({
```

Comment:
We could try using `getPageData` like the fetch above. Pretty sure it takes similar params and means you don't need to import this function or `getAgent`